### ISSUE TITLE
Jira: image attributes

### ISSUE THREAD

**Issue by git4-me** [jgm/pandoc#6212]
The Jira import doesn't recognize images as specified by Atlassian.

Example as taken from https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=all
```
!http://www.host.com/image.gif!
!attached-image.gif!
!image.jpg|thumbnail!
!image.gif|align=right, vspace=4!
```

The html output is:
```
<p><img src="http://www.host.com/image.gif" /><br />
<img src="attached-image.gif" /><br />
!image.jpg|thumbnail!<br />
!image.gif|align=right, vspace=4!</p>
```

As you can see the first two images are properly parsed and recognized. But the later two are not recognized as images.

**Comment by git4-me** [comment]
The Jira import doesn't recognize images as specified by Atlassian.

Example as taken from https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=all
```
!http://www.host.com/image.gif!
!attached-image.gif!
!image.jpg|thumbnail!
!image.gif|align=right, vspace=4!
```

The html output is:
```
<p><img src="http://www.host.com/image.gif" /><br />
<img src="attached-image.gif" /><br />
!image.jpg|thumbnail!<br />
!image.gif|align=right, vspace=4!</p>
```

As you can see the first two images are properly parsed and recognized. But the later two are not recognized as images.

**Comment by tarleb** [comment]
Fixed in jira-wiki-markup-1.1.4.
